src/components/icons/service.icons.ts

```diff
@@ -23,13 +24,31 @@ const getSetOfIcons = async (icons: string[], neededFields: string = '') => {
 }
 
 const updateIcon = async (iconID: string, updateDetails: Object) => {
-  await Model.findOneAndUpdate({ _id: iconID }, updateDetails)
+  const updated = await Model.findOneAndUpdate({ _id: iconID }, updateDetails)
+  return updated
+}
+
+const addSuggested = async (iconName: string, type: suggestionInterfaces.suggestionType, additions: string[]) => {
+  const updateDetails = { $set: {}, $addToSet: {} }
+  if (type === 'tags') {
```

Comment:
can you use the enum here? `suggestionType.tags` and `suggestionType.category`

Reply:
Sure, I will commit the changes.